src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -50,6 +50,7 @@
 /**
  * Unit test for IndentationCheck.
  */
+// -@cs[JavaNCSS] testing phase.
 public class IndentationCheckTest extends AbstractModuleTestSupport {
```

Comment:
Please move this suppression to external suppression file.
https://github.com/checkstyle/checkstyle/blob/master/config/suppressions.xml

Reply:
Please add few cases with violations in both Inputs to show that we are keeping doing validation.